setup.py

```diff
@@ -17,7 +17,8 @@
     platforms='any',
     install_requires=[
         'django',
-        'opentracing>=1.1,<1.2'
+        'opentracing>=1.1,<1.3',
+        'future',
```

Comment:
In other projects under the `opentracing` repositories we have used `six` instead of `futures` - any reason to do so?

IIRC, `futures` is a richer layer, going as far as back-porting whole features or components between Python 2 and 3 - and I think we don't really need that here. `six` is smaller and well tested, and should be just fine for our purposes (specially that we **only** need to use a `iteritems` bridge).

Reply:
Also - do you **need** OT 1.3.0 for this specific patch? I'd rather have it as a separate PR, as it's a different change (one valid on its own ;) )